INSTALL.md

```diff
@@ -581,6 +582,49 @@ To build,
 cmake .. -DBUILD_WITH_QT6=ON -DWITH_QTWEBKIT=OFF -DWITH_QTWEBENGINE=ON
 ```
 
+## 3.12. Building on Linux with vcpkg
+
+With [vcpkg](https://github.com/microsoft/vcpkg/) you can develop QGIS using
+Qt6 on a Linux system.
+
+Install and initialize vcpkg
+
+```sh
+export VCPKG_ROOT=$HOME/.vcpkg
+wget https://aka.ms/vcpkg-init.sh -O /tmp/vcpkg-init.sh
```

Comment:
Is this really the official link for this tool? I'd say it's highly dubious advise to get people to download and run a shell script off a dubious looking domain :upside_down_face:

Reply:
The domain is tried to Microsoft and the official documentation mentions it. @m-kuhn did suggest to add a link to that documentation above, that'll allow ppl to see it referred by the vcpkg project itself.

Reply:
How about we remove this then, and just defer to https://github.com/microsoft/vcpkg-tool/blob/main/README.md#installuseremove ?